app/src/main/res/values-pl/arrays.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+	<string-array name="pref_sync_status_entries">
+		<item>W Posiadaniu</item>
+		<item>Kiedyś W Posiadaniu</item>
+		<item>Na Sprzedaż</item>
+		<item>Chcę Odkupić</item>
+		<item>Chcę Kupić</item>
+		<item>Lista Życzeń</item>
+		<item>Chcę Zagrać</item>
+		<item>Zamówione</item>
+		<item>Zagrane</item>
+		<item>Ocenione</item>
+		<item>Skomentowane</item>
+		<item>Mam Części</item>
+		<item>Szukam Części</item>
+	</string-array>
+	<string-array name="pref_sync_status_values">
```

Comment:
Remove this array from the file. The values will be supplied by the default arrays in `values`.